src/query/v2/requests.hpp

```diff
@@ -403,7 +403,7 @@ struct ExpandOneRequest {
   std::vector<std::string> vertex_expressions;
   std::vector<std::string> edge_expressions;
 
-  std::optional<std::vector<OrderBy>> order_by;
+  std::vector<OrderBy> order_by;
```

Comment:
But that isn't necessary since the expression itself contains either node or edge evaluation (i.e. `MG_SYMBOL_NODE` and `MG_SYMBOL_NODE`), and when passed in evaluator it is clear if it is vertex or edge order by